src/test/java/pages/HomePage.java

```diff
@@ -3,11 +3,27 @@
 import org.openqa.selenium.By;
 import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+
+import java.util.ArrayList;
+import java.util.List;
 
 public class HomePage extends BasePage {
 
     By userAvatarIcon = By.cssSelector("img.avatar");
 
+    By songsPageLink = By.cssSelector("ul.menu a.songs");
+
+    @FindBy(xpath = "//i[@data-testid='play-next-btn']")
+    public WebElement playNextBtn;
+    @FindBy(xpath = "//span[@data-testid='play-btn']")
+    public WebElement playBtn;
+    @FindBy(xpath = "//div[@data-testid='sound-bar-play']")
+    public WebElement soundbar;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```